.github/workflows/test-cluster-set-up.yml

```diff
@@ -45,13 +45,13 @@ jobs:
                 fi
             done
             
-            .github/scripts/userdata.sh "$distribution_type" "$security"
+            release-tools/scripts/userdata.sh "$distribution_type" "$security"
             ls -ltr
 
             echo "Creating $stackName stack"
 
             aws cloudformation create-stack --stack-name $stackName \
-            --template-body file://.github/templates/odfe-testing-cluster-template.json \
+            --template-body file://release-tools/scripts/templates/odfe-testing-cluster-template.json \
```

Comment:
release-tool/templates you give the wrong dir

Reply:
Corrected